collections/internet-censorship-circumventions/index.md

```diff
@@ -0,0 +1,13 @@
+---
+items:
+ - Dreamacro/clash
+ - Psiphon-Inc/psiphon
+ - getlantern/lantern
+ - shadowsocks/shadowsocks
+ - trojan-gfw/trojan
+ - v2fly/v2ray-core
+ - v2ray/v2ray-core
```

Comment:
This repository looks to be pointing to `v2fly/v2ray-core`, can we remove this entry?
```suggestion
```

Reply:
Initially v2ray/v2ray-core, after the disappearance of the original author of this project, the community rebuilt project v2fly/v2ray-core.

Reply:
So, would you agree that the newer, maintained project/org of v2fly/v2ray-core should be the one listed?